src/gauges.rs

```diff
@@ -28,7 +31,7 @@ pub struct PrometheusGauges {
     pub current_epoch: IntGauge,
     pub current_epoch_first_slot: IntGauge,
     pub current_epoch_last_slot: IntGauge,
-    pub leader_slots: IntGaugeVec,
+    pub leader_slots: IntCounterVec,
```

Comment:
Does the dashboard json config file have to be updated accordingly?

Reply:
I can look into that a bit later, possibly tomorrow.